docs/rules/no-force-true.md

```diff
@@ -0,0 +1,37 @@
+# disallow using of &#39;force: true&#39; option (no-force-true)
+
+Using force:true on inputs appears to be confusing rather than helpful.
+It usually silence the actual problem instead of providing a way to overcome it.
+See [Cypress Core Concepts](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Forcing).
+
+
+## Rule Details
+
+This rule aims to disallow using of force option on inputs.
```

Comment:
This is inaccurate to say 'on inputs', this would effect any DOM element you can click or type. 

```suggestion
This rule aims to disallow using the `force` option for [`.click()`](https://on.cypress.io/click) and [`.type()`](https://on.cypress.io/type).
```

Reply:
fixed